LayoutTests/fast/scrolling/mac/scrollbars/scrollbars-controller-type.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ MockScrollbarsEnabled=false AsyncOverflowScrollingEnabled=true ] -->
+<html>
+<head>
+    <style>
+        body {
+            height: 1000px;
+        }
+        .scroller {
+            width: 100px;
+            height: 100px;
+            border: 1px solid black;
+            overflow: scroll;
+        }
+        .contents {
+            width: 100%;
+            height: 200%;
+        }
+s
+    </style>
+    <script src="../../../../resources/js-test.js"></script>
+    <script src="../../../../resources/ui-helper.js"></script>
+    
+    <script>
+        jsTestIsAsync = true;
+
+        if (window.internals)
+            internals.setUsesOverlayScrollbars(true);
+
+        async function doTest()
+        {
+            description('Test hover over the overlay scrollbar of a overflow: scroll in an SVG foreign object');
```

Comment:
It would be good if the test name reflected what it was testing (like this) rather than some internal detail about the controller type.

Reply:
Updated this description as it wasn't accurate. We need to test this internal detail as there isn't a good way of detecting a user visible issue from it.